composer.json

```diff
@@ -26,8 +26,8 @@
     ],
     "require": {
         "php": "^7.1",
-        "doctrine/mongodb-odm": "^1.0",
-        "doctrine/mongodb-odm-bundle": "^3.0",
+        "doctrine/mongodb-odm": "^1.0 || ^2.0",
+        "doctrine/mongodb-odm-bundle": "^4.0",
```

Comment:
Why are we not using `^3.0 || ^4.0` here? No way the 7.1 build works otherwise…

Reply:
hello @greg0ire you're right, I updated the composer.json. thank u for the review